students/barecki_andrzej/lesson_02_flow_control/L6_The_number_of_elements_equal_to_the_maximum.py

```diff
@@ -0,0 +1,16 @@
+# A sequence consists of integer numbers and ends with the number 0. Determine how many elements of this sequence are
+#  equal to its largest element.
+
+input_number = int(input())
```

Comment:
jeżeli będziesz wychodził przez break (patrz komentarz poniżej) to wtedy input możesz przenieść do wnętrzna pętli, dzieki temu nie będziesz musiał powtarzać kodu